transforms/comp_cubemap_parallax.py

```diff
@@ -28,12 +28,7 @@ def comp_cubemap_parallax(ctx: Context):
         diff = maxs - mins
 
         # ensure bounding box has volume
-        if diff[0] == 0.0:
-            diff[0] = 1.0
-        if diff[1] == 0.0:
-            diff[1] = 1.0
-        if diff[2] == 0.0:
-            diff[2] = 1.0
+        diff.max((1.0, 1.0, 1.0))
```

Comment:
This is technically different if the bounds are specified in the wrong order, but making it look wrong is probably enough of a hint to the mapper that they messed up their numbers.

Reply:
Ah true, `<` not `==`.